saw-core/src/Verifier/SAW/Grammar.y

```diff
@@ -337,8 +337,8 @@ mkTupleProj t _ =
      return (badTerm (pos t))
 
 -- | Parse a term as a dotted list of strings
-parseModuleName :: Term -> Maybe [String]
-parseModuleName (RecordProj t str) = (++ [Text.unpack str]) <$> parseModuleName t
+parseModuleName :: Term -> Maybe [Text]
+parseModuleName (RecordProj t str) = (++ [str]) <$> parseModuleName t
```

Comment:
The name `str` here seems to suggest that it's a `String`, but it's really `Text`. Perhaps pick a different variable name.

Reply:
Done in f1d548cdc5d48ed995cb2bb0747d51bfb9979ad6.